.circleci/config.yml

```diff
@@ -2,14 +2,24 @@ version: 2.1
 
 orbs:
   # Using 1.0 of our Orbs means it will use the latest 1.0.x version from https://github.com/wordpress-mobile/circleci-orbs
-  ios: wordpress-mobile/ios@1.0 
+  ios: wordpress-mobile/ios@dev:pod-repo-push
```

Comment:
⚠️  We need to revert this to `@1.0` once https://github.com/wordpress-mobile/circleci-orbs/pull/69 is merged and we made a new `1.0.x` tag to publish the orb.

Reply:
Nice.

Reply:
What was your experience working with orbs from a branch? 

Whenever I tried that, I always experienced a lag between pushing changes to the orb branch and CircleCI applying it when fetching the orb from that branch.

Reply:
Ah, I'm glad I'm not alone in this 😅 . Yep, experienced some lag too:

 - Tbh at first I didn't realize that even for `@dev:branch` references we still needed the orb to be published to the CircleCI registry. So I first expected the orb to be immediately available right after a push, while in practice you need to wait for the orb repo's own CircleCI workflow to finish (and to complete the "publish to orb registry" job) before you can hope use it (so just like for an new official release of the orb).
 - Then even once I realized this, I find that there was still some lag/delay between the orb supposedly being published by CircleCI and it being available for the other projects/workflows. I attributed this to both some propagation delay, and to CircleCI's status being yellow that day (apparently they were on maintenance while I was working on this).
 - I also didn't always monitor the orb's own CircleCI workflow page, so at some point in my development process I didn't realize I had a syntax error in my orb code (I used `<<` in my bash script without escaping it), so I kept pushing changes to the orb, but not seeing any changes when using that orb's `@dev:pod-repo-push` branch from WPAuth's own CircleCI job, and was quite puzzled until I realized that the publication of the orb to the registry failed, which is why my changes seemed ignored and were never released

All that is indeed a bit tedious to work with and can make you lose quite some time when you need to work in parallel on the orb and the repo testing it…